data/ketentuan-pengguna.json

```diff
@@ -36,7 +36,7 @@
       },
       {
         "title":"Penyelesaian Perselisihan",
-        "description":"<ol class='text-[#424242] dark:text-[#AAB0B7]' style='list-style-type:decimal;margin-left:15px;'><li>Anda sepakat untuk menyelesaikan perselisihan dalam mengenai pelaksanaan Ketentuan ini secara musyawarah dan mufakat.</li><li>Apabila musyawarah dan mufakat tidak tercapai dalam waktu 30 (tiga puluh) hari kalender, maka Anda setuju bahwa penyelesaian perselisihan tersebut akan diteruskan dan diselesaikan di Kantor Pengadilan Negeri Bandung.</li></ol></p>"
+        "description":"<ol class='text-[#424242] dark:text-[#AAB0B7]' style='list-style-type:decimal;margin-left:15px;'><li><p style='margin-left:2px;'>Anda sepakat untuk menyelesaikan perselisihan dalam mengenai pelaksanaan Ketentuan ini secara musyawarah dan mufakat.</p></li><li><p style='margin-left:2px;'>Apabila musyawarah dan mufakat tidak tercapai dalam waktu 30 (tiga puluh) hari kalender, maka Anda setuju bahwa penyelesaian perselisihan tersebut akan diteruskan dan diselesaikan di Kantor Pengadilan Negeri Bandung.</p></li></ol></p>"
```

Comment:
minor:

if you intend to use v-html you can style the element using a class so you don't have to write styles on the data itself 

Example:

```js
<template>
  <div v-html="data" class="my-class">
</template>

<style scoped>
  .my-class p { ... }
<style>
```